### ISSUE TITLE
Connecting issues NDI Polaris spectra/stealth S7 localizer to Plus and NDI toolkit

### ISSUE THREAD

**Issue by arterialflow** [PlusToolkit/PlusLib#1081]
Hi, I have an old stealth S7 with dead cpu but  localizer still works (tested on another machine. I'd like to try Plus/Slicer. When I tried on my Win10 laptop, it fails to connect using SCUs USB (suspect the USB driver not avail?) and with USB-serial FTDI to the 9 pin dsub on SCU. A label beneath the conn on SCU says RS422.  Would it be worth while trialing an RS422 FTDI? Should I try breaking out the serial (RS232?) on the lemo cable to localizer itself?  many thanks

**Comment by lassoan** [comment]
Yes, connecting with an USB to RS 422 FTDI adapter makes sense (or get an RS 232/422 converter and connect to the computer using a very commonly available USB to serial (RS 232) adapter).

**Comment by arterialflow** [comment]
OK thanks , I'll try the dtech usb-rs422 from amazon since it has LED tx/rx indicators.

**Comment by arterialflow** [comment]
I think I have success on RS422. 

 Plus Server Launcher & fcal  both show Connection Successfull!.  RX and TX leds flash continuously on my RS422 FTDI which I suspect is SCU reporting data but I don't have a tool(8700339), stylus or reference. Is that why I don't see any activity or marker display?? I plan to 3D print these. 
NDI toobox aslo shows connected but gives <<ERROR016BC2
